src/components/Lab/LabObservationQuestion.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+import { Label } from "@/components/ui/label";
+
+import ValueSetSelect from "@/components/Questionnaire/ValueSetSelect";
+
+import { LabObservation } from "@/types/emr/observation";
+import { Code } from "@/types/questionnaire/code";
+
+import { LabObservationItem } from "./LabObservationItem";
+
+interface LabObservationQuestionProps {
+  question: string;
+  observations: LabObservation[];
+  setObservations: React.Dispatch<React.SetStateAction<LabObservation[]>>;
+  disabled?: boolean;
+}
+export const LabObservationQuestion: React.FC<LabObservationQuestionProps> = ({
+  question,
+  observations,
+  setObservations,
```

Comment:
use shadcn form component and react hook form to maintain the states and enforce validation